requirements.txt

```diff
@@ -5,15 +5,15 @@ click-pathlib==2020.3.13.0
 cryptography==42.0.4
 immutabledict==2.2.1
 jwcrypto==1.5.6
-prettytable==3.4.1
-prompt-toolkit==3.0.31
+prettytable==3.7.0
+prompt-toolkit==3.0.40
 PyNaCl==1.5.0
 pytz==2022.2.1
-PyYAML==6.0.1
-questionary==1.10.0
-requests==2.32.0
-rich==12.5.1
+PyYAML==6.0.2
+questionary==1.10.0 # not mantained since 2018, can break anytime
+requests==2.32.2
+rich==13.6.0
```

Comment:
This is a major bump, include (in the PR description) if (or if not) there is anything that could possibly break e.g.  having being deprecated, just for documentation purposes.

Reply:
I linked in the description the release notes for the libraries affected. All the packages that are updated is because they mention there either deprecations or dropping/adding support for the python versions affected